app/src/main/java/com/example/cocktails/Cocktails.kt

```diff
@@ -43,11 +42,15 @@ class Cocktails : Application() {
         mActiveTypeFilters = this.getSharedPreferences(TYPE_FILTERS, Context.MODE_PRIVATE)
         mStorageRef = FirebaseStorage.getInstance().reference
         mFirstTimeModeSP = this.getSharedPreferences(FIRST_TIME_MODE, Context.MODE_PRIVATE)
+        mUserInputs=this.getSharedPreferences(USER_INPUT,Context.MODE_PRIVATE)
 
         if(mUserId.isEmpty()){
             val userId= mFirstTimeModeSP.getString(USER_ID_KEY_SP,"")
             if (userId.isNullOrEmpty()){
                 mUserId = java.util.UUID.randomUUID().toString()
+                while (checkIfUserIdAlreadyExist(mUserId)){
```

Comment:
niceeeee

Reply:
I don't think it's work well. So don't remove the todo. I will check this deeply tomorrow :)